README.md

```diff
@@ -1,43 +1,43 @@
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/3ba6416fd11d41fdaf281e7dab6042dc)](https://www.codacy.com/app/philwhiles/census-contact-centre-service?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ONSdigital/census-contact-centre-service&amp;utm_campaign=Badge_Grade)

```

Comment:
Don't understand why github has flagged so much change since you only changed one line didn't you ? !
No issue with the README, just confused about what github is showing!

Reply:
I only changed that line as an insignificant change because I needed to trigger travis. So I don't know what github is flagging here either!